kiwi/config/functions.sh

```diff
@@ -552,18 +552,17 @@ function baseStripUnusedLibs {
     # /.../
     # add exceptions
     # ----
-    while [ ! -z "$1" ];do
+    for j in $1; do
```

Comment:
Thanks for the fix this is much better. While we are on it why not use a more meaningful variable e.g replace 'j' with 'libname' or similar. Thoughts ?

Reply:
yes I am fine with using better variable names. I'll change both `$j` and `$i`. I used `$j` just because the other inner loop was using `$i`. Next one would have been `$k`, `$l` ... :grinning: 

Gonna change it now